pipeline/outputs/stackdriver.md

```diff
@@ -189,3 +190,5 @@ Do following check:
 Stackdriver officially supports a [logging agent based on Fluentd](https://cloud.google.com/logging/docs/agent).
 
 We plan to support some [special fields in structured payloads](https://cloud.google.com/logging/docs/agent/configuration#special-fields). Use cases of special fields is [here](./stackdriver_special_fields.md).
+
+[StackdriverSpecialFields]: ./stackdriver_special_fields.md#log-entry-fields
```

Comment:
Hi @lecaros, this should make it so the links aren't broken. How did you test it?